schema/properties/source.json

```diff
@@ -0,0 +1,33 @@
+{
+    "schemaId": "properties/source",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "The source of a characteristic. This could be an article, a simulation on Exabyte, an external simulation, etc.",
```

Comment:
We should replace "characteristic" with "property" everywhere in this PR. The former is outdated language.